server/server/src/file_index_repo.rs

```diff
@@ -172,7 +189,8 @@ pub async fn change_document_content_version(
                 content_version =
                     (CASE WHEN NOT old.deleted AND old.metadata_version = $2 AND NOT old.is_folder
                     THEN CAST(EXTRACT(EPOCH FROM NOW()) * 1000 AS BIGINT)
-                    ELSE old.content_version END)
+                    ELSE old.content_version END),
+                document_size = $3
```

Comment:
this should be a conditional set just like the content version one is. yeah, you have to copy and paste that whole `CASE` thing (the 3 lines above) because SQL doesn't have good abstractions. otherwise document size will be updated even if this fails (for instance because their version is out of date)

Reply:
a915883